parser-typechecker/src/Unison/Syntax/DeclPrinter.hs

```diff
@@ -73,9 +74,9 @@ prettyGADT env ctorType r name dd =
         (DD.constructors' dd)
   where
     constructor (n, (_, _, t)) =
-      prettyPattern env ctorType name (ConstructorReference r n)
+      prettyPattern (PPED.unsuffixifiedPPE env) ctorType name (ConstructorReference r n)
```

Comment:
Passing in a `PrettyPrintEnvDecl` (which has both suffixed and unsuffixed PPEs) was part of the fix for #3239.